Use incorrect sts endpoint when using China regions

Use aws credential like below,

```yaml
    - name: Configure AWS credentials
      uses: aws-actions/configure-aws-credentials@v1
      with:
        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
        aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
        aws-region: cn-northwest-1
```
Got below error,
```
Run aws-actions/configure-aws-credentials@v1
##[error]Inaccessible host: `sts.cn-northwest-1.amazonaws.com'. This service may not be available in the `cn-northwest-1' region.
```

According to [the doc of AWS China region](https://docs.amazonaws.cn/en_us/general/latest/gr/cnnorthwest_region.html), the correct endpoint of STS in China region is `sts.cn-northwest-1.amazonaws.com.cn`. Of course, `cn-northwest-1` will be replaced as `cn-north-1` in BJS region.

--- a/index.js
+++ b/index.js
@@ -1,7 +1,6 @@
 const core = require('@actions/core');
 const aws = require('aws-sdk');
 const assert = require('assert');
-const util = require('util');
 
 // The max time that a GitHub action is allowed to run is 6 hours.
 // That seems like a reasonable default to use if no role duration is defined.
@@ -131,10 +130,9 @@ async function exportAccountId(maskAccountId, region) {
 }
 
 function getStsClient(region) {
-  const endpoint = util.format('https://sts.%s.amazonaws.com', region);
   return new aws.STS({
     region,
-    endpoint,
+    stsRegionalEndpoints: 'regional',
     customUserAgent: USER_AGENT
   });
 }

